commercetools-models/src/main/java/io/sphere/sdk/categories/commands/updateactions/SetAssetCustomField.java

```diff
@@ -0,0 +1,32 @@
+package io.sphere.sdk.categories.commands.updateactions;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.types.customupdateactions.SetCustomFieldBase;
+
+/**
+ * This action sets, overwrites or removes any existing custom field for an existing Asset.
+ *
+ * @see io.sphere.sdk.types.Custom
+ */
+public final class SetAssetCustomField extends SetCustomFieldBase<Category> {
+    private final String assetId;
+
+    private SetAssetCustomField(final String assetId, final String name, final JsonNode value) {
+        super("setAssetCustomField", name, value);
+        this.assetId = assetId;
+    }
+
+    public String getAssetId() {
+        return assetId;
+    }
+
+    public static SetAssetCustomField usingJson(final String assetId, final String name, final JsonNode value) {
```

Comment:
I also thought about `ofJsonValue` - it lead me to think the `final JsonNode value` is the first param, and not the `final String assetId`, but perhaps that's just me, coming from the Swifty background. Let's use `ofJsonValue`, cause I don't think we'll figure out anything better here ☺️